### DISCUSSION TITLE
Have a |slug filter

### DISCUSSION THREAD

**Discussion by maxstrebel** [craftcms/cms#14389] [Ideas] [answered]
Very often I find myself writing `text|lower|ascii|kebab` or something similar, in order to sanitize text for the use in an id (`<div id="{{ anchorText|lower|ascii|kebab }}">`).

Unfortunately this is a rabbit hole, as you find yourself removing special characters, etc.

While I understand that the |slug filter might be confusing due to its differences to the native slug functionality (https://github.com/craftcms/cms/issues/12515), would you consider to have `|slug` and maybe `|craft_slug` (for the lack of a better name)?

**Comment by WHITE-developer** [comment, marked as answer]
@maxstrebel there is a dedicated `|id` filter https://craftcms.com/docs/4.x/dev/filters.html#id

**Reply by maxstrebel** [reply]
Thank you, so much!
